torch/testing/_internal/distributed/rpc/jit/rpc_test.py

```diff
@@ -1084,4 +1084,13 @@ def test_async_function_wrong_return_type(self):
             rpc.rpc_sync(
                 worker_name((self.rank + 1) % self.world_size),
                 async_wrong_type
-            )
\ No newline at end of file
+            )
+
+    @dist_init
+    def test_async_function_wrong_decorator_order(self):
+        with self.assertRaises(RuntimeError):
```

Comment:
What error is raised here? Is this something that `torch.jit.script` complains about since there is another decorator?

Reply:
`torch.jit.script` failed to compile this function. It says it does not recognize `rpc`. Let me add the error to the comments. 

The reason for not checking the error string is to avoid making JIT error handling depending on RPC tests.